deploy/docker-compose/configs/task-manager.json

```diff
@@ -0,0 +1,115 @@
+{
+  "WorkloadManagerDatabase": {
+    "ConnectionString": "mongodb://root:rootpassword@mongodb:27017",
+    "DatabaseName": "WorkloadManager",
+    "WorkflowCollectionName": "Workflows",
+    "WorkflowInstanceCollectionName": "WorkflowInstances",
+    "PayloadCollectionName": "Payloads",
+    "TaskDispatchEventCollectionName": "TaskDispatchEvents"
+  },
+  "WorkflowManager": {
+    "endpointSettings": {
+      "defaultPageSize": 10,
+      "maxPageSize": 1000000
+    },
+    "TaskManager": {
+      "concurrency": 1,
+      "taskTimeoutMinutes": 60,
+      "plug-ins": {
+        "argo": "Monai.Deploy.WorkflowManager.TaskManager.Argo.ArgoPlugin, Monai.Deploy.WorkflowManager.TaskManager.Argo",
+        "docker": "Monai.Deploy.WorkflowManager.TaskManager.Docker.DockerPlugin, Monai.Deploy.WorkflowManager.TaskManager.Docker",
+        "aide_clinical_review": "Monai.Deploy.WorkflowManager.TaskManager.AideClinicalReview.AideClinicalReviewPlugin, Monai.Deploy.WorkflowManager.TaskManager.AideClinicalReview",
```

Comment:
This one also? There is no harm unless someone messes with the workflow def, or use a copied def for the full version.